plate_mapper/plate_mapper.py

```diff
@@ -54,8 +55,9 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None, special_f=None):
         Sample name list file
     special_f : file object (optional)
         Special sample definition file
+    empty : bool (optional)
```

Comment:
Here "empty" is a well without sample. The test data files in the repository demonstrates empty-allowed vs empty-free mapping files.